src/languages/arcade.js

```diff
@@ -75,7 +249,8 @@ export default function(hljs) {
   ]);
 
   return {
-    name: 'ArcGIS Arcade',
+    aliases: [ 'arcade' ],
+    case_insensitive: true,
```

Comment:
The big changes here were keywords and case insensitive, right?    That and relevance?  If I back out all the tiny changes (that I asked you to change back) and just copied over the keywords and case sensitivity and took a look at the relevance issues myself, would that achieve the goals here?

I know you wanted to pull this out so you were probably tuning this a bit more to your liking, but now that it's staying...

Reply:
@joshgoebel that's correct the only real changes are the addition of new keywords in the recent language update and case insensitive. The relevance was changed only to get the tests to pass so it has no functional significance to the parsing. So I'm fine with working out what you think is best so we can get this in the next update.

Reply:
@joshgoebel I just restored `name`

not sure what you want to do about `relevance`, do you want me to revert all of those changes?

Reply:
You don't need to do anything further, I'll take a pass from scratch and tag you on the keyword/sesnativity change.